presto-native-execution/Makefile

```diff
@@ -113,7 +113,7 @@ centos-container:			#: Build the linux container for CircleCi
 linux-container:
 	rm -rf /tmp/docker && \
 	mkdir -p /tmp/docker && \
-	cp scripts/setup-$(CONTAINER_NAME).sh scripts/$(CONTAINER_NAME)-container.dockfile velox/scripts/setup-helper-functions.sh /tmp/docker && \
+	cp scripts/setup-$(CONTAINER_NAME).sh scripts/$(CONTAINER_NAME)-container.dockfile velox/scripts/setup-centos8.sh velox/scripts/setup-helper-functions.sh /tmp/docker && \
```

Comment:
should this be velox/scripts/setup-${CONTAINER_NAME} ? ..

Reply:
In prestissimo the setup script is called `setup-centos.sh` but in velox it is `setup-centos8.sh`. So it shouldn't be CONTAINER_NAME because that would be `centos` and there would be no equivalent script.
However, what if this is `ubuntu`? But also `setup-${CONTAINER_NAME}8` wouldn't work....

Reply:
Will fix!

Reply:
I made a comment about this but I deleted a duplikcate comment so I guess it deleted both?
Anyway I said that in prestissimo the script is called `setup-centos.sh` while in Velox it is `setup-centos8.sh`.

The CONTAINER_NAME is set to `centos`. So we'd need `setup-${CONTAINER_NAME}8` here. But this doesn't work if the container is `ubuntu`. If it is only CentOS then we are ok.

Reply:
I am removing this target as part of the https://github.com/prestodb/presto/pull/21929.
I am thinking we can use `if-else` for now :).